x/concentrated-liquidity/lp_test.go

```diff
@@ -162,6 +168,128 @@ var (
 			amount1Expected: DefaultAmt1Expected,
 		},
 	}
+
+	amountZeroExtendedRange = osmomath.NewInt(1_000_000).Add(osmomath.NewInt(10).ToLegacyDec().PowerMut(18).TruncateInt())
+	amountOneExtendedRange  = osmomath.NewInt(50).Mul(osmomath.NewInt(1_000_000))
+
+	// coins that initiliaze initial spot price to be in the
+	// extended range
+	// 50 * 10^6 / 10^6 * 10^18 = 50 * 10^-18
+	coinsExtendedFullRangePrice = sdk.NewCoins(
+		sdk.NewCoin(ETH, amountZeroExtendedRange),
+		sdk.NewCoin(USDC, amountOneExtendedRange),
+	)
+
+	// configures the lp test case for testing full range extension
+	// of the min spot price.
+	//
+	// What this does:
+	// - Disable irrelevant parameters such as spread rewards, slippage
+	// protection and coin funding
+	// - Sets tick spacing to 1
+	// - Calculate expected liquidity
+	// - Caclulate expected amounts from liquidity
+	//
+	// Assumes that internal math functions are implemented correctly.
+	// roundUp is true for creating position and false for withdrawing position.
+	estimateLPMigrationCaseResults = func(lpTest lpTest, roundUp bool) lpTest {
+		// Note that the min sqrt price must be computed from the min tick since that's
+		// the assumption made in LP logic.
+		lowerSqrtPrice, _ := math.TickToSqrtPrice(lpTest.lowerTick)
+		upperSqrtPrice, _ := math.TickToSqrtPrice(lpTest.upperTick)
```

Comment:
Noting the circular reliance on internal functions to compute expected values – in the past, bugs related to ticks/price precision tended to show up in these internal functions, so we should mentally flag that low-level edge cases might be missed here

Reply:
Followed-up in DM. Will resolve